module-bluetooth/Bluetooth/CommandHandler.cpp

```diff
@@ -51,15 +51,15 @@ namespace bluetooth
         case bluetooth::Command::StopScan:
             return stopScan();
         case bluetooth::Command::Pair:
-            return pair(command.getDevice());
+            return pair(std::get<Devicei>(command.getData()));
```

Comment:
can this logic be moved to generic command? i.e. it would just return variant on `getParsed` it would require knowing all command types in the base class. ( Could be moved to "parser" class though)

Reply:
the base class has to know all types because of `DataVariant`. I think it's possible to do a generic command and return there any optional data, but it makes another layer of decision making - either switch/case or overloaded methods for each data type